v2/pkg/options/options.go

```diff
@@ -148,7 +160,10 @@ func processMenus(appoptions *App) {
 	switch runtime.GOOS {
 	case "darwin":
 		if appoptions.Menu == nil {
-			appoptions.Menu = defaultMacMenu
+			appoptions.Menu = menu.NewMenuFromItems(
```

Comment:
Was this change intended? Just wondering how it affected the behaviour. Perhaps we should remove the default variables if they are causing an issue?

Reply:
Yeah it was intended but not needed for the fix. It was just that after removing the default handling there was a `default.go` file with just the content of the `defaultMacMenu`. So I thought we could remove that and get rid of that almost empty `default.go` file.